tardis/tests/test_rpacket.py

```diff
@@ -0,0 +1,146 @@
+import os
+from sys import float_info as floats
+import random
+from ctypes import CDLL, c_double
+
+import pytest
+import numpy as np
+
+from tardis import __path__ as path
+
+test_path = os.path.join(path[0], 'montecarlo', 'montecarlo.so')
+MAX_FLOAT, MIN_FLOAT = floats[0], floats[3]
+MAX_INT = 10000
+MIN_INT = -10000
+
+tests = CDLL(test_path)
+SIZE = 10
+
+
+# Testing functions with float(C double) valued parameters
+np.random.seed(1)
```

Comment:
I think you should set one random seed at the top and then be done with it. The sequence of random numbers will then always be the same for all tests.